src/main/java/ru/skypro/homework/dto/Comments.java

```diff
@@ -0,0 +1,23 @@
+package ru.skypro.homework.dto;
+
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Schema(description = "список комментариев")
+public class Comments {
+    @Schema(description = "количество комментариев")
+    private Integer count;
+    @Schema(description = "комментарии")
+    private List<CommentDto> result;
```

Comment:
```suggestion
    private List<CommentDto> results;
```

Также просмотрите и убедитесь, что у вас больше нет расхождений со спекой